README.md

```diff
@@ -18,7 +18,7 @@ pip install -r requirements.txt
 ```bash
 flask --app app run --debug --no-reload
 ```
-#### Running in production mode:
-```bash
-TODO
-```
+> [!WARNING]
+> Currently the app won't destroy the Docker container after the user closes the tab.</br>
+> To manually stop the container, run:<br/>```docker ps``` to get the container ID,</br>
+> and then ```docker stop <container_id>``` to stop the container.
```

Comment:
use `docker kill <container_id>` instead. 
Sometime Omen hangs and `docker stop` does not work immediately.

Reply:
Fixed at [1f05830](https://github.com/omen-osdev/omen-demo/pull/6/commits/1f05830017e5a48058940999cda00343b29a9bb0)